robotiq_description/urdf/robotiq_sim_gripper.ros2_control.xacro

```diff
@@ -14,56 +15,81 @@
+                    <state_interface name="position"/>
+                    <state_interface name="velocity"/>
+                </xacro:unless>
             </joint>
             <joint name="robotiq_85_left_inner_knuckle_joint">
                 <param name="mimic">robotiq_85_left_knuckle_joint</param>
                 <param name="multiplier">1</param>
-                <command_interface name="position" />
-                <state_interface name="position" />
-                <state_interface name="velocity" />
+                <xacro:unless value="${sim_ignition}">
+                    <command_interface name="position"/>
+                    <state_interface name="position"/>
+                    <state_interface name="velocity"/>
+                </xacro:unless>
             </joint>
             <joint name="robotiq_85_right_inner_knuckle_joint">
                 <param name="mimic">robotiq_85_left_knuckle_joint</param>
                 <param name="multiplier">-1</param>
-                <command_interface name="position" />
-                <state_interface name="position" />
-                <state_interface name="velocity" />
+                <xacro:unless value="${sim_ignition}">
+                    <command_interface name="position"/>
+                    <state_interface name="position"/>
+                    <state_interface name="velocity"/>
+                </xacro:unless>
             </joint>
             <joint name="robotiq_85_left_finger_tip_joint">
                 <param name="mimic">robotiq_85_left_knuckle_joint</param>
                 <param name="multiplier">-1</param>
-                <command_interface name="position" />
-                <state_interface name="position" />
-                <state_interface name="velocity" />
+                <xacro:unless value="${sim_ignition}">
+                    <command_interface name="position"/>
+                    <state_interface name="position"/>
+                    <state_interface name="velocity"/>
+                </xacro:unless>
             </joint>
             <joint name="robotiq_85_right_finger_tip_joint">
                 <param name="mimic">robotiq_85_left_knuckle_joint</param>
                 <param name="multiplier">1</param>
-                <command_interface name="position" />
-                <state_interface name="position" />
-                <state_interface name="velocity" />
+                <xacro:unless value="${sim_ignition}">
+                    <command_interface name="position"/>
+                    <state_interface name="position"/>
+                    <state_interface name="velocity"/>
+                </xacro:unless>
             </joint>
-            <gpio name="reactivate_gripper">
-                <command_interface name="reactivate_gripper_cmd" />
-                <command_interface name="reactivate_gripper_response" />
-            </gpio>
+            <!-- Not active for Ignition -->
```

Comment:
Pretty sure this also has issues in Isaac and is not needed. I'll test and let you know.

Reply:
If I keep these tags on, it makes Ignition segfault lol

Reply:
Did you remove [this line](https://github.com/PickNikRobotics/isaac_moveit_studio_config/blob/main/isaac_ur5e_base_config/config/base_config.yaml#L128) from the Isaac robot config so it doesn't complain when you start the system?

Reply:
Will do!